velox/functions/sparksql/tests/JsonArrayLengthTest.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/sparksql/tests/SparkFunctionBaseTest.h"
+
+using namespace facebook::velox::test;
+
+namespace facebook::velox::functions::sparksql::test {
+namespace {
+
+class JsonArrayLengthTest : public SparkFunctionBaseTest {
+ protected:
+  VectorPtr jsonArrayLength(const std::string& json) {
+    auto varcharVector = makeFlatVector<std::string>({json});
+    return evaluate("json_array_length(c0)", makeRowVector({varcharVector}));
```

Comment:
Perhaps use 'evaluateOnce<int32_t>' to get the output for simplicity.

Reply:
done